### PR TITLE
amazon ses instead of sendgrid (CA-222 and CA-269)

### PR DESCRIPTION
**Pull request by logophobia**

### CODE REVIEW

File: CollAction/Startup.cs [hunk 1]
```diff
@@ -84,9 +84,10 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<ISmsSender, AuthMessageSender>();
             services.Configure<AuthMessageSenderOptions>(options =>
             {
-                options.FromName = Configuration["FromName"];
                 options.FromAddress = Configuration["FromAddress"];
-                options.SendGridKey = Configuration["SendGridKey"];
+                options.Region = RegionEndpoint.EnumerableAllRegions.First(r => r.SystemName == Configuration["SesRegion"]);
+                options.SesAwsAccessKey = Configuration["SesAwsAccessKey"];
```
Comment (stewartmatheson) [inline CollAction/Startup.cs]: Does this get set from the environment?

File: env_collaction.template [hunk 2]
```diff
@@ -3,10 +3,10 @@ DbHost=db
 DbUser=postgres
 DbPort=5432
 Db=CollActionDb
-FromName=
 FromAddress=your@email
-SendGridUser=
-SendGridKey=
+SesAwsAccessKey=
+SesAwsAccessKeyId=
+SesRegion=
```
Comment (marcbeermann) [inline env_collaction.template]: Maybe update the 'Getting CollAction running locally' with the Ses configurations.

File: CollAction/Startup.cs (hunk 1 above)
Comment (logophobia) [inline CollAction/Startup.cs]: Either the environment, a config file or user secrets (in debug).

File: env_collaction.template (hunk 2 above)
Comment (logophobia) [inline env_collaction.template]: I will.

**Pull request merged**